src/main/java/org/opensearch/neuralsearch/processor/dto/CombineScoresDto.java

```diff
@@ -29,4 +31,6 @@ public class CombineScoresDto {
     private List<QuerySearchResult> querySearchResults;
     @Nullable
     private Sort sort;
+    private int fromValueForSingleShard;
+    private boolean isFetchResultsPresent;
```

Comment:
it doesn't look right to put this field here, it's not related to combination. please find alternative solution